scripts/dict_audit_symbols.py

```
@@ -0,0 +1,299 @@
+            print(f"  - U+{start:04X}-U+{end:04X} ({chr(start)}-{chr(end)}) [Total Freq: {total_freq}]")
+
+    # Format for convert-builtin-fonts.sh
+    print("\n--- Copy-paste for convert-builtin-fonts.sh ---")
+    
+    unicode_strs = []
+    for start, end in suggested_ranges:
+        if start == end:
+            unicode_strs.append(f"U+{start:04X}")
+        else:
+            unicode_strs.append(f"U+{start:04X}-U+{end:04X}")
+    print(f"Append to IPA_UNICODES:")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)